### PR TITLE
Improving kubectl get output

### PR DESCRIPTION
**Pull request by Ritikaa96**
Signed-off-by: Ritikaa96 <ritika@india.nec.com

#### What this PR does / why we need it:
 Adding namespace flag & examples
kubectl get --help states the following even though we can use --namespace to get details for other namespace resources.
```shell
Display one or many resources.

 Prints a table of the most important information about the specified resources.
You can filter the list using a label selector and the --selector flag. If the
desired resource type is namespaced you will only see results in your current
namespace unless you pass --all-namespaces.
...
```
The description should mention `--namespace ` as this can show objects/resources from different namespace as well

#### Which issue(s) this PR fixes:

Fixes #https://github.com/kubernetes/kubectl/issues/1555

#### Does this PR introduce a user-facing change?
NONE

#### Does this PR introduce a user-facing change?
NONE

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
NONE

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ var (
 		Prints a table of the most important information about the specified resources.
 		You can filter the list using a label selector and the --selector flag. If the
 		desired resource type is namespaced you will only see results in your current
-		namespace unless you pass --all-namespaces.
+		namespace unless you pass --all-namespaces or --namespace(for specific namespace)
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:95]: What about?
```suggestion
		namespace by passing --namespace(for specific namespace) unless you pass --all-namespaces.
```

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 2]
```diff
@@ -132,7 +132,13 @@ var (
...
 
 		# List the 'status' subresource for a single pod
-		kubectl get pod web-pod-13je7 --subresource status`))
+		kubectl get pod web-pod-13je7 --subresource status
+
+		# List all deployments in namespace - backend
+		kubectl get deployments.apps --namespace backend
+
+		# List all pods existing in all namespaces
+		kubectl get pods --all-namspaces`))
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:141]:
```suggestion
		kubectl get pods --all-namespaces`))
```

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 3]
```diff
@@ -132,7 +132,13 @@ var (
 		kubectl get rc/web service/frontend pods/web-pod-13je7
 
 		# List the 'status' subresource for a single pod
-		kubectl get pod web-pod-13je7 --subresource status`))
+		kubectl get pod web-pod-13je7 --subresource status
+
+		# List all deployments in namespace - backend
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:137]:
```suggestion
		# List all deployments in namespace 'backend'
```

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go (hunk 2 above)
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:141]: oh!! how did i miss that.

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go (hunk 1 above)
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:95]: looks better . I'll make the change.

Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:95]: or better..
```go
If the desired resource type is namespaced you will only see results in other namespaces by passing --namespace(for specific namespace) or --all-namespaces.
```

**Comment by Ritikaa96** [comment]
Made some changes PTAL @ardaguclu

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 4]
```diff
@@ -91,8 +91,8 @@ var (
 
 		Prints a table of the most important information about the specified resources.
 		You can filter the list using a label selector and the --selector flag. If the
-		desired resource type is namespaced you will only see results in your current
-		namespace unless you pass --all-namespaces.
+		desired resource type is namespaced you will only see results in other namespaces
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:94]: or current namespace if user doesn't specify none of them?

**Comment by Ritikaa96** [comment]
Hi , is there anything to add in this PR as the PR is awaiting reviews from long.

**Comment by ardaguclu** [comment]
> Hi , is there anything to add in this PR as the PR is awaiting reviews from long.

This https://github.com/kubernetes/kubernetes/pull/123440/files#r1502075458 comment hasn't been resolved yet.

**Comment by Ritikaa96** [comment]
> > Hi , is there anything to add in this PR as the PR is awaiting reviews from long.
> 
> This https://github.com/kubernetes/kubernetes/pull/123440/files#r1502075458 comment hasn't been resolved yet.

Hi @ardaguclu I have made some changes. PTAL. Does it look good or i should make another change?

**Comment by ardaguclu** [comment]
Thank you
/lgtm
/approve
/triage accepted
/priority backlog

**Comment by ardaguclu** [comment]
this is only doc change;
/retest

**Comment by Ritikaa96** [comment]
Seems like all the test have passed . Waiting for @dims review .
Thanks a lot @ardaguclu !!

**Pull request merged**